pkgs/applications/misc/3dslicer-binary/default.nix

```diff
@@ -0,0 +1,61 @@
+{stdenv, pkgs}:
```

Comment:
please avoid using pkgs, and instead import each dependency individually.

This allows for things like `.override()` to work correctly